### ISSUE TITLE
pt estimate for T5s in parseT5 function is deprecated

### ISSUE THREAD

**Issue by jkguiang** [SegmentLinking/TrackLooper#249]
The pT for T5s in the output NTuple [are computed](https://github.com/jkguiang/TrackLooper/blob/master/code/core/write_sdl_ntuple.cc#L410) as follows:
```cpp
float pt = quintupletsInGPU.innerRadius[quintupletIndex] * kRinv1GeVf;
...
ana.tx->pushbackToBranch<float>("t5_pt", pt);
```
However, the [pT estimate](https://github.com/jkguiang/TrackLooper/blob/master/code/core/write_sdl_ntuple.cc#L1041-L1042) in ```parseT5``` is just the average of the T3 pT's (which is somewhat questionable per #214):
```cpp
// Compute pt estimates from inner and outer triplets
const float ptAv_in = abs(dr_in * k2Rinv1GeVf / sin((betaIn_in + betaOut_in) / 2.));
const float ptAv_out = abs(dr_out * k2Rinv1GeVf / sin((betaIn_out + betaOut_out) / 2.));

// T5 pt is average of the two pt estimates
const float pt = (ptAv_in + ptAv_out) / 2.;
```
It's my understanding that the pT as computed for the T5s in the output NTuple are "more correct," and that both the pTs from the output NTuple and ```parseT5``` are used and therefore giving inconsistent results. I am working on a PR (will include link here when available) that will add some additional branches for the GNN NTuple, so I can make the fix in that PR as well if desired.

**Comment by VourMa** [comment]
@jkguiang Is this still an issue? Did you deal with it in any of your PRs,as mentioned in the description?

**Comment by VourMa** [comment]
Closed by #388
